grafana/README.md

```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
 * Login as admin/admin
 * Change admin password
 * Add Prometheus datasource
-* Import all 4 dashboards 
+* Import all 5 dashboards 
   * PostgreSQL.json
   * PostgreSQLDetails.json
   * BloatDetails.json
   * CRUD_Details.json
+  * TableSize_Details.json
+
+
+### API Import
+
+It is possible it import these graphs through the "import" HTTP API using the following curl command to add some required wrapper information to each json blob. 
+```
+curl --user admin:admin -S "http://localhost:3000/api/dashboards/import" -X POST -H 'Content-Type: application/json;charset=UTF-8' --data-binary "{  \"dashboard\" : $(cat PostgreSQL.json) , \"overwrite\":true, \"inputs\":[  {  \"name\":\"DS_PROMETHEUS\", \"type\":\"datasource\", \"pluginId\":\"prometheus\", \"value\":\"PROMETHEUS\" } ] }"
+```
+You will likely have to edit the following parts of the above command:
+
+ * Username
```

Comment:
I suggest " Replace admin:admin with user:password with access to rest api ex: grafana admin"